draft-kinnear-webtransport-http2.md

```diff
@@ -506,6 +506,39 @@ on the WebTransport Connect Stream associated with the CONNECT request and reset
 all WebTransport Streams associated with the session.
 
 
+# Transport Properties
+
+The WebTransport framework {{?I-D.vvv-webtransport-overview}} defines a set of
+optional transport properties that clients can use to determine the presence of
+features which might allow additional optimizations beyond the common set of
+properties available via all WebTransport protocols. Http2Transport supports the
```

Comment:
I would replace "Http2Transport supports the following properties" with something like "Below are details about whether Http2Transport supports these properties"

Reply:
(Because I find it odd to say we support properties and then say we don't)

Reply:
Ah thank you, was looking for a way to say that -- was bothering me too.

Reply:
Changed in df81a40